packages/create-svelte/templates/default/.gitignore

```diff
@@ -1,12 +1,17 @@
-.DS_Store
-node_modules
-/build
-/.svelte-kit
-/package
-.env
-.env.*
-!.env.example
-.vercel
 .output
+.svelte-kit
+.vercel
+
+build
+node_modules
+package
```

Comment:
I think it could lead to trouble to remove the leading `/`. If the user has something like `src/packages/package` as we do in SvelteKit it would end up getting ignored, which isn't what we want